libs/wingc/src/parser.rs

```diff
@@ -791,15 +791,23 @@ impl<'s> Parser<'s> {
 	}
 
 	fn build_bring_statement(&self, statement_node: &Node) -> DiagnosticResult<StmtKind> {
-		let module_name_node = self.get_child_field(&statement_node, "module_name")?;
+		let Some(module_name_node) = statement_node.child_by_field_name("module_name") else {
+			return self.with_error("Expected built-in module or \"external module\"", &statement_node.child(statement_node.child_count() - 1).unwrap_or(*statement_node));
```

Comment:
Not totally happy with this wording, very open to suggestions

Reply:
See my suggestion above although not sure if it's better.